src/fleet_app/client_wrapper.cr

```diff
@@ -9,10 +11,14 @@ module FleetApp
         @client = client.not_nil!
       else
         # Instantiate a client based on the environment passed in
-        if @environment == "production"
+        case @environment
+        when "production"
           @client = FleetApp::Client.new
-        else
+        when "staging", "sandbox"
           @client = FleetApp::Client.new(FleetAppClient::SANDBOX_HOST)
+        else
+          # use localhost for "test" and "development" environments
+          @client = FleetApp::Client.new(FleetAppClient::DEVELOPMENT_HOST)
```

Comment:
To send brooce commands during development simply set games-site LUCKY_ENV to sandbox